.github/workflows/python-package.yml

```
@@ -183,11 +183,11 @@ jobs:
 
   deploy:
     needs: isolated_tests
-    if: startsWith(github.ref, 'refs/tags/')
+    if: github.ref_type == 'tag'
     runs-on: ubuntu-latest
     environment:
-      name: deploy
-      url: https://pypi.org/project/delocate/
+      name: pypi
+      url: ${{ github.ref_name }}
```

The URL should still be full
```suggestion
      url: https://pypi.org/project/delocate/${{ github.ref_name }}
```